src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/LearnerChangeOfCircumstance/LearnerChangeOfCircumstanceCommandHandler.cs

```diff
@@ -33,7 +36,22 @@ public async Task Handle(LearnerChangeOfCircumstanceCommand command, Cancellatio
 
             var learner = await _learnerDomainRepository.GetOrCreate(incentive);
 
-            await incentive.SetChangeOfCircumstances(learner, _collectionCalendarService);
+            if (learner.SubmissionData.SubmissionFound)
+            {
+                if (learner.SubmissionData.LearningData.StartDate.HasValue)
+                {
+                    incentive.SetStartDateChangeOfCircumstance(learner.SubmissionData.LearningData.StartDate.Value);
+
+                    if (incentive.StartDateHasChanged())
+                    {
+                        await incentive.CalculateEarnings(_incentivePaymentProfilesService, _collectionCalendarService);
+                    }
```

Comment:
Wouldn't this be better in the SetStartDateChangeOfCircumstance method?

Reply:
> Wouldn't this be better in the SetStartDateChangeOfCircumstance method?

I thought about doing that that but the command handler is the use case and orchestrator for this process.  The domain entity is to model the business logic.  So I suppose the question is, is this a specific use case or an intrinsic business rule?